program-runtime/src/invoke_context.rs

```diff
@@ -598,6 +598,11 @@ impl<'a> InvokeContext<'a> {
         &self.current_compute_budget
     }
 
+    /// Get this invocation's compute meter.
+    pub fn get_compute_meter(&self) -> Ref<u64> {
```

Comment:
Is this used anywhere except the new test?

Reply:
Nope! Are you going to suggest `get_compute_meter_for_tests`? Or remove?

Reply:
At the least, let's add the `"dev-context-only-utils"` attribute. Or maybe remove?

Reply:
I've added a commit to remove this, rather than add `dev-context-only-utils`, which doesn't exist for program-runtime.